packages/go_router/pubspec.yaml

```diff
@@ -1,7 +1,7 @@
 name: go_router
 description: A declarative router for Flutter based on Navigation 2 supporting
   deep linking, data-driven routes and more
-version: 5.1.2
+version: 5.1.4
```

Comment:
5.1.3?

Reply:
With the newly merged PRs, I'll make it 5.1.6